certified-connectors/Aletheia/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1651 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Aletheia",
+    "description": "Aletheia is an online web service providing financial data through an easily accessible HTTP REST API. Access stock data, SEC filings, Insider Trades, earnings calls, and more.",
+    "version": "1.0",
+    "contact": {
+      "name": "Tim Hanewich",
```

Comment:
We usually advocate for the `contact` metadata to not contain PII, as this information will be user-facing. Any chance this could instead point to a general support email within the Aletheia domain that you have? And the name could then be changed to something like `Aletheia Support`.

Reply:
Doug - thanks for the feedback. I just pushed an update to these details. Let me know if I am missing anything else. Thank you.